tests/rados/test_scrub_chunk_max.py

```diff
@@ -0,0 +1,173 @@
+
+        assert mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
+
+        init_time, _ = installer_node.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+        rados_obj.run_deep_scrub(pool=pool_name)
+
+        start_time = datetime.datetime.now()
+        while datetime.datetime.now() <= start_time + datetime.timedelta(seconds=600):
+            pool_pg_dump = rados_obj.get_ceph_pg_dump(pg_id=pool_pgid)
+            if rhbuild and rhbuild.split(".")[0] >= "5":
+                objects_scrubbed = int(pool_pg_dump["objects_scrubbed"])
+                log.info(f"Objects scrubbed: {objects_scrubbed} | Target: {objs}")
+                if (objects_scrubbed - org_objects_scrubbed) >= objs:
+                    end_time, _ = installer_node.exec_command(
+                        cmd="sudo date '+%Y-%m-%d %H:%M:%S'"
+                    )
+                    break
+                else:
+                    log.info(
+                        f"Objects scrubbed are less than target ({objs}). "
+                        f"Sleeping for 30 secs"
+                    )
+                    time.sleep(30)
+            if (
+                pool_pg_dump["last_deep_scrub"] != "0'0"
+                and pool_pg_dump["state"] == "active+clean"
+            ):
+                log.info(f"Deep-scrub completed, pg state: {pool_pg_dump['state']}")
+                break
+            else:
+                log.info(
+                    f"deep-scrub is yet to complete, pg state: {pool_pg_dump['state']}. "
+                    f"Sleeping for 30 secs"
+                )
+                time.sleep(30)
+        else:
+            log.error(
+                f"All {objs} objects in pool {pool_name} couldn't get scrubbed within 10 mins"
+            )
+            return 1
+
+        log.info(f"Final last_deep_scrub: {pool_pg_dump['last_deep_scrub']}")
+        log.info(
+            f"Final last_deep_scrub_stamp: {pool_pg_dump['last_deep_scrub_stamp']}"
+        )
+        log.info(f"Final objects_scrubbed: {pool_pg_dump['objects_scrubbed']}")
+
+        scrub_logs = rados_obj.get_journalctl_log(
+            start_time=init_time,
+            end_time=end_time,
+            daemon_type="osd",
+            daemon_id=primary_osd,
+        )
+
+        for line in scrub_logs.splitlines():
+            if f"pg[{pool_id}.0(" in line and "be_scan_list" in line:
+                pg_scrub_log_list.append(line)
+
+        pg_scrub_log = "\n".join(pg_scrub_log_list)
+        log.debug(
+            f"\n ==========================================================================="
+            f"\n Scrub log entries: \n {pg_scrub_log}"
+            f"\n ==========================================================================="
+        )
+
+        for i in range(3):
+            log.info(f"Verifying existence of log entry 'be_scan_list ({i}/3'")
+            assert f"be_scan_list ({i}/3" in pg_scrub_log
```

Comment:
Add a modulus operation to get the remainder of the no_obj to verify this. Also mention that it should specifically be a non-divisible no of the "org_scrub_chunk_max" var.

Add a comment as well.

Reply:
done